docs/concepts/configuration/assign-pod-node.md

```diff
@@ -190,13 +190,7 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
-In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limited set of topology keys are allowed.
-By default the allowed topology keys are:
-
-* `kubernetes.io/hostname`
-* `failure-domain.beta.kubernetes.io/zone`
-* `failure-domain.beta.kubernetes.io/region`
+The `topologyKey` can be any legal label value.
```

Comment:
I'd prefer that we still highlight that it is suggested to use those following three `topologyKey` for better performance.

How about:

In principle, the `topologyKey` can be any legal label value. However, for performance reasons, it is suggested to use the following three keys:
		
* `kubernetes.io/hostname`		
* `failure-domain.beta.kubernetes.io/zone`		
* `failure-domain.beta.kubernetes.io/region`